src/web/app/components/question-edit-form/question-edit-form.component.ts

```diff
@@ -246,15 +246,26 @@ export class QuestionEditFormComponent implements OnInit {
     if (allowedRecipientTypes.indexOf(recipientType) === -1) {
       newRecipientType = allowedRecipientTypes[0];
     }
-    this.triggerModelChangeBatch({
-      giverType,
-      recipientType: newRecipientType,
-      commonVisibilitySettingName: 'Please select a visibility option',
-      isUsingOtherVisibilitySetting: false,
-      showResponsesTo: [],
-      showGiverNameTo: [],
-      showRecipientNameTo: [],
-    });
+    if (this.model.giverType === giverType && this.model.recipientType === newRecipientType) {
+      // do not reset the visibility settings if reverting feedback path to preset template provided
+      if (this.model.isUsingOtherFeedbackPath) {
```

Comment:
It is perhaps not a good practice to have two such if's as they create an arrow-antipattern. You might want to just do 
```suggestion
    if (this.model.giverType === giverType && this.model.recipientType === newRecipientType
      && this.model.isUsingOtherFeedbackPath) {
```
in which the precedence of the booleans achieves the exact same effect.

Reply:
Wrong logic here. Shouldn't omit the path where the function does nothing